### ISSUE TITLE
Cross version compilation on OS X

### ISSUE THREAD

**Issue by Chris911** [rake-compiler/rake-compiler#91]
Hi again,

I know issues are normally not for support but I don't know where to ask. I got rake-compiler to work on OS X with some specific frameworks but I just figured out it the generated gem only works for 1 version of Ruby (the one I compiled with). I've been trying to figure out to build the gem to work with multiple version of Ruby but not luck so far.

I installed the rake-compiler dev box (vagrant) but since I need some OS X librairies with the gem that option didn't work since the box runs linux and those librairies are not available. 

I have all the developer tools installed on my dev machine but couldn't get any of the `rake cross [...]` commands to work. What am I missing?

**tl;dr** I want to build a gem for multiple ruby version (not multiple platforms) on my mac. What are the steps needed to get this to work?

Thanks for your time!
